module7_selenium/src/test/java/ICanWinTestMain.java

```diff
@@ -0,0 +1,25 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import page.PastebinStartPage;
+
+public class ICanWinTestMain {
+    private WebDriver driver;
+
+    @BeforeMethod(alwaysRun = true)
+    public void browserSetup() {
+        driver = new ChromeDriver();
+    }
+
+    @Test(description = "Test for task 'I can win'")
+    public void createPageWithText() {
```

Comment:
When I run this method, I get:
```
java.lang.IllegalStateException: The path to the driver executable must be set by the webdriver.chrome.driver system property; for more information, see https://github.com/SeleniumHQ/selenium/wiki/ChromeDriver. The latest version can be downloaded from http://chromedriver.storage.googleapis.com/index.html
```
Please create setup scripts to run your code
or
add instructions to run your code
or
use https://github.com/bonigarcia/webdrivermanager